components/dashboard/src/data/projects/fetchers.ts

```diff
@@ -64,9 +64,11 @@ export const useFetchProjects = ({ teamId, userId }: UseFetchProjectsArgs) => {
 
         // Sort projects by latest prebuild first
         projects.sort((p0: Project, p1: Project): number => {
-            return dayjs(latestPrebuilds.get(p1.id)?.info?.startedAt || "1970-01-01").diff(
-                dayjs(latestPrebuilds.get(p0.id)?.info?.startedAt || "1970-01-01"),
-            );
+            // use latest prebuild start time, then fallback to project creation if no prebuild
```

Comment:
Could we do this sorting on the backend instead? It would mean we get a consistent behaviour for all clients of the API, not just dashboard.

Reply:
I'd love it if we could do that, along with not needing the client to fan out and make an api call for each project's latest build. I'm assuming we'd need to see if we could include a latest build for each project in our query (would be helpful if we included it in the response, or had an option to).

Reply:
We can start with just doing the sorting backend, and then see where we can go from there. Would you like to take a stab at it? Happy to pair as well.

Reply:
Somewhat related, I think we could stop loading _all projects_ initially to speed up the initial load. We can use the pagination on the Public API to load just a page, and render it before we load the rest. Ideally, we wouldn't even load the next pages unless you click "next page" or search